app/auto-eligibility/register-your-interest/process-eligible-sbi.js

```diff
@@ -21,6 +69,29 @@ const processEligibleSbi = async (customer) => {
     }
     await customerDbTable.updateWaitingUpdatedAt(customer.sbi, customer.crn)
     await emailNotifier.sendWaitingListEmail(customer.businessEmail)
+    await raiseEvent({

```

Comment:
Possible duplicate of app/auto-eligibility/register-your-interest/process-eligible-sbi.js line 15

Reply:
true but this duplication is due to feature toggle and will be removed once feature toggle is removed

now it's for simplicity